docs/docs/use-the-factory-orchestrator-api.md

```diff
@@ -19,16 +19,16 @@ await Client.RunExecutable(@"%windir%\system32\ping.exe");
 ```
 
 ## Using FactoryOrchestratorClient in PowerShell
-The FactoryOrchestratorClient PowerShell module is available in FactoryOrchestratorClient.psd1.
+The FactoryOrchestratorClient PowerShell module is available in Microsoft.FactoryOrchestrator.Client.psd1.
 
-To use the PowerShell module, import FactoryOrchestratorClient.psd1 and then use the New-FactoryOrchestratorClient cmdlet to create a FactoryOrchestratorClient instance. The PowerShell FactoryOrchestratorClient instance exposes the exact same APIs as the C# FactoryOrchestratorClient class. However, unlike FactoryOrchestratorClient and FactoryOrchestratorUWPClient C# classes, all calls are synchronous.
+To use the PowerShell module, import Microsoft.FactoryOrchestrator.Client.psd1 and then use the New-FactoryOrchestratorClient cmdlet to create a FactoryOrchestratorClient instance. The PowerShell FactoryOrchestratorClient instance exposes the exact same APIs as the C# FactoryOrchestratorClient class. However, unlike FactoryOrchestratorClient and FactoryOrchestratorUWPClient C# classes, all calls are synchronous.

```

Comment:
do we have api docs for tje cmdlet? no problem if not, but perhaps a future task/issue.

Reply:
https://microsoft.github.io/FactoryOrchestrator/use-the-factory-orchestrator-api/#using-factoryorchestratorclient-in-powershell

Just dis ^^. It's pretty simple / mirrors C# but a bit more of an explainer seems nice.